app/models/user.rb

```diff
@@ -29,4 +29,9 @@ class User < ApplicationRecord
   validates_each :first_name, :last_name, :address do |record, attr, value|
     record.errors.add(attr) if value =~ /\A[a-z]/
   end
+
+  def full_name
+    [first_name, last_name].select(&:present?).join(' ').titleize
```

Comment:
`"#{first_name} #{last_name}"` should be enough

Reply:
and add test for this method

Reply:
wrote test